### ISSUE TITLE
Check on mandatory sport type assignment not done on league creation

### ISSUE THREAD

**Issue by teddyfresco** [diddipoeler/sportsmanagement#330]
Saving a league, a check on mandatory sport type assignment is not done, and a record is set on db without sport type.

**Comment by diddipoeler** [comment]
please try again.

**Comment by teddyfresco** [comment]
Just tried with JSM downloaded now, on J3.6.2.
Still it does not make the check and leagues without sport are saved.

**Comment by diddipoeler** [comment]
please try again.

**Comment by teddyfresco** [comment]
just tried again (emptied db and installed from github link) but still does not work, I can save a league also without short name and country.
I also think this could be a minor, secondary bug, a big hurdle is events assignment #335, maybe we could contact some external dev to help on that? I am willing to contribute to pay, in case...

**Comment by diddipoeler** [comment]
i can´t create a league without name,short name, sporttype and country.

**Comment by teddyfresco** [comment]
Dumped db, uninstalled jsm, then installed from github, on Joomla 3.6.2 and PHP 5.5.36 
Previously didn't check without name as well, only without short name, sporttype and country.
Now, tried twice emptying db and uninstalling, I saved successfully without name,short name, sporttype and country. I wrote nothing on the page and save was successful.
Sad news is that I can save also clubs and seasons without anything inside. Items are correctly listed, but empty, on respective pages.
